scss/_accordion.scss

```diff
@@ -0,0 +1,146 @@
+//
+// Base styles
+//
+
+.accordion-button {
+  @include font-size($font-size-base);
+  display: flex;
+  align-items: baseline;
+  width: 100%;
+  padding: $accordion-button-padding-y $accordion-button-padding-x;
+  color: $accordion-button-color;
+  text-align: left;
+  background-color: $accordion-button-bg;
+  border: $accordion-border-width solid $accordion-border-color;
+  border-bottom: none;
+  @include border-radius(0);
+
+  &[aria-expanded="true"] {
```

Comment:
I would prefer classes here. In general we use the aria labels for accessibility and classes for theming.